modules/Emails/EmailsController.php

```diff
@@ -135,6 +135,7 @@ public function action_ComposeView()
     public function action_send()
     {
         $this->bean = $this->bean->populateBeanFromRequest($this->bean, $_REQUEST);
+        $this->bean->mailbox_id = $_REQUEST['inbound_email_id'];
```

Comment:
Can you please move this into the populateBeanFromRequest? So that we can reuse this code as much as possible.

I have just noticed that this method isn't using the $request variable. We will need to update this.